tests/fixtures/remove_class/assertEqual_in.py

```diff
@@ -6,4 +6,22 @@ def test_you(self):
 class TestAssertNotEqual(TestCase):
 
     def test_you(self):
-        self.assertNotEqual(abc, 'xxx')
+        """
```

Comment:
This change removes the test for "assertNotEqual". Please add an new test fixture. Simply adding new `*_in.py` and `*_out.py` files shoudl suffice.